### PR TITLE
remove func_conifg.train.xxx

### PR DESCRIPTION
**Pull request by doombeaker**
test_2d_gpu_variable.py test_TestMultiInputGrad.py test_TestReshape4KeepHeaderOnly.py test_activations.py test_all_reduce_group.py test_batch_gather.py test_batch_normalization.py test_bias_add.py test_binary_elementwise_ops.py test_broadcast_normal.py test_broadcast_to_compatible_with.py test_cast.py test_clip_by_value.py test_compat_conv2d.py test_concat.py test_deconv2d.py test_dropout.py test_dynamic_reshape.py test_expand_dims.py test_gather.py test_gather_nd.py test_get_variable.py test_inplace.py test_l2_normalize.py test_layer_norm.py test_layers_conv2d.py test_leaky_relu.py test_matmul.py test_multiply.py test_nn_conv2d.py test_non_distribute_optimizer.py test_oneflow_export.py test_pool.py test_prelu.py test_reduce_mean.py test_reduce_opsV2.py test_reshapeV2.py test_scalar_by_tensor_ops.py test_scatter_nd.py test_slice_v2.py test_smooth_l1_loss.py test_softmax.py test_softmax_cross_entropy.py test_sparse_cross_entropy.py test_sparse_softmax_cross_entropy.py test_square.py test_squeeze.py test_transpose.py test_two_stage_reduce.py test_unsorted_batch_segment_sum.py test_unsorted_segment_sum_fw_bw.py test_upsample.py test_watch_diff.py test_where.py

remove all flow.add_loss and func_config.primary_lr and naiv_conf. use flow.optimizer.SGD(lr_scheduler, momentum=0).minimize(loss) instead

### CODE REVIEW

**Comment by doombeaker** [comment]
#3372 has been merged which is same as this PR
